aep/general/0005/aep.md

```diff
@@ -0,0 +1,84 @@
+# Designing an API
+
+This AEP serves as a high-level guide on designing an API that adheres to the
+AEPs.
+
+## Process summary
+
+1. enumerate the user case would like your API to satisfy.
```

Comment:
```suggestion
1. Enumerate the use cases you would like your API to satisfy.
```

I suggest capitalizing all of these.

Also, is "and add them to your API" on each of the next three lines necessary?

Reply:
I was adding the "add them to your API" since step 1 doesn't require them to do that. But we can try removing it.